sites/website/src/docs/community/engineering-process/release-planning.md

```diff
@@ -0,0 +1,61 @@
+
+### Team Review & Priority adjustment
+
+Now that all the issues are prepared, the team reviews each issue starting from the highest priority ones. We discuss the issues and agree on the priority at this point. Sometimes we make adjustments to the suggested individual priorities. After discussing each issue the work is summarized in the quarterly plan document, where we double check our alignment to team mission, pillars, and OKRs.
+
+### Capacity Planning
+
+We usually reserve only 50% of the team capacity for the defined quarterly work. The reason is that we will be getting a lot of incoming feedback throughout the year and we need to allocate time for handling this feedback throughout the year. So we calculate the capacity of the team in weeks for the upcoming year and use half of the final number later in this process.
+
+### Define the cut line
+
+At this point we have all the candidate issues that we think are worth considering for the upcoming quarter. This number is quite large, so the teams usually won't have enough capacity to handle all this.
+We start stack ranking issues so the most important work remains on the top of the list. We then draw the cut line and that defines the rough list of things the team will work on during the upcoming quarter. If there are any issues we think may be at risk, we indicate those as well.
+
+## Release Cadence
+
+Our quarterly planning enables us to tackle larger projects and strategic work while our weekly planning enables us to be adaptive and responsive to the changing needs of the community and project. As a result, each week typically produces PRs to the project, which triggers releases.
+
+### Minor and Patch Releases
+
+The majority of the work coming from the quarterly planning process and the weekly planning process results in minor features and bug fixes. Once a PR for a `minor` or `patch` change is merged to master it will be automatically published in the nightly release. Releases are published Sunday through Thursday evenings (Pacific). Unless there is an emergency need, we avoid releasing on Fridays or the weekend (manual intervention required by us).
+
+### Major Releases
+
+Major releases signal breaking changes in libraries. We strive to have only 1-2 major releases per year, so that we don't destabilize the community. This means that we will often delay certain issues from one quarter to the next in order to explicitly batch the work together into a single breaking-change release. Sometimes component libraries undergo significant visual refreshes. In this case, even though there is no breaking API change, we bump the major version to avoid surprises for the community.
+
+## Acknowledgements
+
+Many thanks to the .NET team for sharing their policies and practices upon which this document was based.
```

Comment:
I think you could remove the duplicate statement made above, and just keep this one.  Also, with a link to the .NET team.